frontend/src/components/HelpDrawer.vue

```diff
@@ -93,10 +84,9 @@ export default defineComponent({
     };
 
     onMounted(() => {
-      event.on(EventType.EVENT_HELP, showHelp);
-    });
-    onUnmounted(() => {
-      event.off(EventType.EVENT_HELP);
+      helpStore.$subscribe((_, state) => {
```

Comment:
Use `watch` instead of `$subscribe`. 

* https://stackoverflow.com/a/71937507

When user click the close button in the drawer, we might need to set the `currHelpId` to `undefined/null`. It makes more sense in store level not UI level.

Reply:
PTAL